server/neptune/workflows/internal/deploy/revision/queue/deployer.go

```diff
@@ -127,14 +130,32 @@ func (p *Deployer) updateCheckRun(ctx workflow.Context, deployRequest terraformW
 	ctx = workflow.WithRetryPolicy(ctx, temporal.RetryPolicy{
 		MaximumAttempts: UpdateCheckRunRetryCount,
 	})
-	err := workflow.ExecuteActivity(ctx, p.Activities.GithubUpdateCheckRun, activities.UpdateCheckRunRequest{
+
+	request := notifier.GithubCheckRunRequest{
 		Title:   terraformWorkflow.BuildCheckRunTitle(deployRequest.Root.Name),
 		State:   state,
 		Repo:    deployRequest.Repo,
-		ID:      deployRequest.CheckRunID,
 		Summary: summary,
 		Actions: actions,
-	}).Get(ctx, nil)
+	}
+
+	version := workflow.GetVersion(ctx, version.CacheCheckRunSessions, workflow.DefaultVersion, 1)
+
+	var err error
+	if version == workflow.DefaultVersion {
```

Comment:
since we are versioning in a couple places and using different request objects for different versions, what do you think of adding a method `toUpdateCheckrunActivityRequest()` to `notifier.GithubCheckRunRequest` so that we can just make that call instead of having to populate the struct everytime?

Reply:
The check run id is one thing that would need to be passed in so it's not direct 1:1.  Id rather just keep it separate. This just makes it easier to remove once we have this deployed.